src/test/java/racing/domain/CarTest.java

```diff
@@ -0,0 +1,29 @@
+package racing.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+public class CarTest {
+  private Car car;
+  @BeforeEach
+  void init(){
+    car = new Car(0);
+  }
+
+  @Test
+  void carStartsAtZero(){
+    assertThat(car.getPosition()).isEqualTo(0);
+  }
+
+  @ParameterizedTest
+  @CsvSource({"4,true", "5,true", "0,false", "3,false"})
```

Comment:
```suggestion
  @CsvSource({"4,true", "9,true", "0,false", "3,false"})
```
경계값 테스트 👍 
단, 5보다는 9로 테스트하는 것이 더 확실한 경계값 테스트이지 않을까?